src/main/kotlin/me/spradling/gift/core/conversion/GiftCommitApiConverter.kt

```diff
@@ -0,0 +1,17 @@
+package me.spradling.gift.core.conversion
+
+import me.spradling.gift.core.api.models.Account
+import java.util.stream.Collectors
+
+class GiftCommitApiConverter {
+
+  fun convert(accounts: List<me.spradling.gift.core.database.models.Account>): List<Account> {
+
+    return accounts.stream().map { account: me.spradling.gift.core.database.models.Account ->
```

Comment:
nit: i'm pretty sure you don't have to specify the type of the lambda argument here.